cfgov/core/management/commands/_helpers.py

```diff
@@ -125,7 +131,7 @@ def authenticate_user(self):
         if user:
             if not user.is_active or not user.is_superuser:
                 raise PermissionDenied('This user does not have permissions to'
-                                       + 'perform this operation.')
+                                       'perform this operation.')
```

Comment:
Not related to your change, but isn't there a missing space between these two phrases?